src/helpers/misc/Animal.re

```diff
@@ -323,7 +323,11 @@ let getOrgBadgeImage: t => string =
     // | Tarkus
     // | _ => None
     };
-
+[
```

Comment:
Whoops, apologies, this was me. I was using this as a copy pad for the new graphql backend. It should be deleted.